alvr/dashboard/src/steamvr_launcher/mod.rs

```diff
@@ -156,11 +171,36 @@ impl Launcher {
         if !is_steamvr_running() {
             debug!("SteamVR is dead. Launching...");
 
-            #[cfg(windows)]
-            windows_steamvr::start_steamvr();
-
-            #[cfg(target_os = "linux")]
-            linux_steamvr::start_steamvr();
+            if let Switch::Enabled(steamvr_path) = data_sources::get_read_only_local_session()
+                .settings()
+                .extra
+                .steamvr_launcher
+                .use_steamvr_path
+                .clone()
```

Comment:
It was better not to clone, but it doesn't matter too much for this code as it's not performance sensitive.